### DISCUSSION TITLE
GitBook Lens

### DISCUSSION THREAD

**Discussion by sarahf16** [GitbookIO/community#370] [0. General bugs and questions] [answered]
Is there an issue with GitBook Lens at the moment, as both of my spaces that have it switched on for, it no longer appears in?

**Comment by SamyPesse** [comment, marked as answer]
Can you share a link to your space?

It might be related to a recent improvement we've made to the UI, where you no longer needs to click a tab, but you can directly type a question and press <kbd>Enter</kbd>:

<img width="917" alt="Screenshot 2023-10-17 at 15 33 56" src="https://github.com/GitbookIO/community/assets/845425/fe3c2439-78e6-4f2c-80ae-f19d5531f39f">

**Reply by sarahf16** [reply]
Thank you for your help - I had missed that this had changed!
